lib/plugins/aws/lib/normalizeFiles.js

```diff
@@ -31,6 +31,18 @@ module.exports = {
       }
     });
 
+    // Sort resources and outputs to ensure consistent hashing
+    const sortByKey = ([key], [otherKey]) => key.localeCompare(otherKey);
+
+    normalizedTemplate.Resources = _.fromPairs(
+      Object.entries(normalizedTemplate.Resources).sort(sortByKey)
```

Comment:
Why not use `deepSortObjectByKey'` here?

Reply:
I was wondering about it and I think it makes sense to use it here as well for both Resources and Outputs :+1: